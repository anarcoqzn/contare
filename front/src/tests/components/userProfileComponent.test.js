import UserProfile from '../../components/dasboardComponent/userProfileComponent/userProfileComponent.jsx';
import {shallowMount} from 'enzyme';
import index from '../../services/index';
describe('<userProfileComponent />', () => {
window.location = {reload: jest.fn()}
const login = jest.fn();
const getToken = jest.fn();


it('should render UserProfileComponent correctly', () => {
    const wrapper = shallow(<UserProfile token = {getToken}/>);
    console.log(wrapper.debug());
    expect(wrapper).toMatchSnapshot();
});


});


