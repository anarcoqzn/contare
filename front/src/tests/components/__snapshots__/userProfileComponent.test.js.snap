// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<userProfileComponent /> should render UserProfileComponent correctly 1`] = `<Route />`;

exports[`should render UserProfileComponent correctly 1`] = `<Route />`;
